admin-dev/themes/new-theme/scss/pages/_translation_page.scss

```diff
@@ -1,5 +1,162 @@
+$color-error: #c05c67 !default;
+$color-active: #25b9d7 !default;
+
 .export-translations {
   .export-translations-child {
     padding-left: 50px;
   }
 }
+
+.breadcrumb {
+  small {
+    color: $gray-medium;
+  }
+}
+
+.total-remaining-translations {
+  color: $color-error;
+}
```

Comment:
As it's specific to the translation page, please use a parent selector related to the translation page